### PR TITLE
Add react component for dev login

### PR DESCRIPTION
**Pull request by anjapatel**
This PR adds a React component for a dev login page.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "webpack": "5.75.0",
     "webpack-cli": "5.0.1"
   },
-  "proxy": "https://planz.example.com/",
+  "proxy": "https://planz.bcholmes.org",
```
Comment (bcholmes) [inline client/package.json:28]: Can you restore this back to the example.com address?

File: client/src/page/compositePage.jsx [hunk 2]
```diff
@@ -17,27 +18,28 @@ import SessionEnumerationConfigPage from './tool/sessionEnumerationConfig';
...
+      return <SessionEnumerationConfigPage />;
+    } else if (url.pathname === "/printRoomScheduleConfig.php") {
+      return <PrintRoomScheduleConfigPage />;
+    } else if (url.pathname === "/devLogin.php") {
+      return <DevLoginPage />;
+    } else {
+      return <h4>Unknown Page</h4>;
     }
+  }
 }
```
Comment (bcholmes) [inline client/src/page/compositePage.jsx:43]: It looks like your VS Code is automatically changing the indent to 2 spaces, whereas other code is set to four spaces. That makes the PR a bit noisier than it needs to be.

File: client/package.json (hunk 1 above)
Comment (anjapatel) [reply client/package.json:28]: Sure. So every time a dev wants to login locally, they'll have to update this? I can add a clarifying comment to the component then.

Comment (bcholmes) [reply client/package.json:28]: Yeah. We expect other devs to use different test servers. I think there's a note in the `client/README.md` about changing the proxy entry to run the local server.

**Pull request merged**